### PR TITLE
Allow spaces and hyphens between phone numbers

### PR DESCRIPTION
**Pull request by gnandretta**

### CODE REVIEW

**Comment by cristiandouce** [comment]
LGTM :+1:

File: src/cred/index.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export function setPhoneNumber(lock, value) {
 }
 
 export function validatePhoneNumber(phoneNumber) {
-  const regExp = /^[0-9]{1,14}$/;
+  const regExp = /^[0-9]([0-9 -])*[0-9]$/;
```
Comment (daf-spr) [inline src/cred/index.js]: I think it is the same, and might even be expected in this context, but the regulars expressions are not equivalent, the first one accepts "9", whereas this one requires at least 2 digits "99"

Comment (gnandretta) [inline src/cred/index.js]: You're right. I wanted to reject a phone number ending with a `-` while keeping the regexp straightforward, and that's what I came up with. I think it's safe to assume phone numbers have at least two digits but I'm not sure this is the best validation we can do. Let's wait to have some feedback and then we can improve it.

Comment (daf-spr) [inline src/cred/index.js]: :+1:

**Pull request merged**